core-drawer-panel.html

```diff
@@ -247,7 +247,7 @@
         <content select="[main]"></content>
         <div id="scrim"></div>
         <div id="edgeSwipeOverlay"
-            class$="[[_computeSwipeOverlayClass(narrow, disableEdgeSwipe)]]">
+            hidden$="[[_computeSwipeOverlayHidden(narrow, disableEdgeSwipe)]]">
```

Comment:
I failed to see that HTML5 added a `hidden` attribute that is `display:none;` by default in browsers that implement it (everything but IE10 and below).

http://www.w3.org/TR/html-markup/global-attributes.html#common.attrs.hidden

No need for a `.hidden` that does the same thing.